rpc/client.go

```diff
@@ -26,8 +26,62 @@ import (
 	"github.com/statechannels/go-nitro/types"
 )
 
-// RpcClient is a client for making nitro rpc calls
-type RpcClient struct {
+// RpcClientApi provides various functions to make RPC API calls to a nitro RPC server
+type RpcClientApi interface {
```

Comment:
I originally named the interface `RpcClient` but I was worried `RpcClient` vs `rpcClient` could be confused.